packages/angular/src/lib/index.ts

```diff
@@ -48,3 +48,35 @@ export {
 	ShapeNodeComponent,
 	ShapeNodeModule
 } from './diagrams'
+
+// Republish essential types from core so it's not a required dependency for TypeScript projects
+export type {
+	ChartTabularData,
+	ChartOptions,
+	AlluvialChartOptions,
+	AreaChartOptions,
+	BarChartOptions,
+	BoxplotChartOptions,
+	BubbleChartOptions,
+	BulletChartOptions,
+	ChoroplethChartOptions,
+	CirclePackChartOptions,
+	ComboChartOptions,
+	DonutChartOptions,
+	GaugeChartOptions,
+	HeatmapChartOptions,
+	HistogramChartOptions,
+	LineChartOptions,
+	LollipopChartOptions,
+	MeterChartOptions,
+	PieChartOptions,
+	RadarChartOptions,
+	ScatterChartOptions,
+	StackedAreaChartOptions,
+	TreeChartOptions,
+	TreemapChartOptions,
+	WorldCloudChartOptions as WordCloudChartOptions
```

Comment:
This is non-breaking because WorldCloudChartOptions was not previously exported.